hail/python/hail/methods/impex.py

```diff
@@ -1710,9 +1710,12 @@ def import_plink(bed, bim, fam,
 
 
 @typecheck(path=oneof(str, sequenceof(str)),
+           intervals=nullable(sequenceof(anytype)),
+           filter_intervals=bool,
            _drop_cols=bool,
            _drop_rows=bool)
-def read_matrix_table(path, _drop_cols=False, _drop_rows=False) -> MatrixTable:
+def read_matrix_table(path, intervals=None, filter_intervals=False, _drop_cols=False,
```

Comment:
let's add a `*` between `path` and the rest of the args, so that they become keyword-only args and we can change the order without affecting backwards compatibility.

Reply:
also, if these params are to be public, they need documentation.